plugin.php

```diff
@@ -21,6 +21,8 @@
 require __DIR__ . '/inc/keys/namespace.php';
 require __DIR__ . '/inc/plc/namespace.php';
 require __DIR__ . '/inc/plc/util.php';
-require __DIR__ . '/vendor/autoload.php';
+
+// A composer-based install of this plugin will not have a local vendor/ dir, nor does it need one.
+@include __DIR__ . '/vendor/autoload.php';
```

Comment:
Instead of silencing the include, could we check for the existence of the `FAIR\Beacon\Provider` interface which would return `true` if the global (project-wide) autoload was present?

Something like:

```php
if ( ! interface_exists( FAIR\Beacon\Provider::class ) ) {
	require_once __DIR__ . '/vendor/autoload.php';
}
```